### ISSUE TITLE
Support for starting at the end of the file or suppressing line numbers

### ISSUE THREAD

**Issue by h2cone** [sharkdp/bat#2473]
Hi, First of all great work!

When I try to view from the end of a very large input file, I entering `+G` in the pager, but it is blocked for a long time, then I tried `export BAT_PAGER="less -RF -n"` which also did not work. After checking the documentation, I found `--line-range <N:M>`, but I need to know the number of lines first. Maybe starting from the end of the file or suppressing the number of lines would be more efficient?

**Comment by haixuanTao** [comment]
I agree that this is a good feature to have as I'm trying to use Bat for viewing logs.

**Comment by andriy-git** [comment]
I would also love to see `less +G` function in bat.
